packages/placeholders/src/solid-faker.js

```diff
@@ -0,0 +1,139 @@
+// Minimal utility for generating Lorem Ipsum random / pseudo-random text
+export default class SolidFaker {
+  constructor(seedValue) {
```

Comment:
Love that you used just JS.

Could you please use JSDocs in the whole class, so that we can create a `d.ts` file in the end?

Reply:
Could you please generate the types already and export them as done in `theming`? :) So that we have them in Git and easily releasable?

Reply:
done!